README.md

```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
 1. Close the current hero-display session
 2. Repeat steps 1-3 & 5 from [First install](#first-install)
 
+
+## HERO1
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
+- **hero1-battery-conversion**: Publishing a BatteryState message from the custom Toyota messages
+- **hero1-monitor.service**: HERO version of `hsrb_monitor` from `hsrb_bringup`
+- **hero1-ntpdate.service**: Service to sync time with servers
+- **hero1-phone-ui.service**: Service to always have the webserver running
+- **hero1-roscore.service**: Service containing the roscore
+- **hero1-speech1-vboxautostart-service**: Slightly changed version of vboxautostart
+- **hero1-speech2-vboxballoonctrl-service**: Slightly changed version of vboxballoonctrl
+- **hero1-speech3-vboxdrv**: Slightly changed version of vboxdrv
+- **hero1-speech4-vboxweb-service**: Slightly changed version of vboxweb
+- **hero1-speech5-windows-speech**: Service to start the Windows virtualbox
+
 ## HERO2
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
+- **hero2-battery-manager**: Running a battery manager to let the robot speak in case of a low battery level
+- **hero2-openpose**: Running OpenPose
```

Comment:
Why does this service always run?

Reply:
It shouldn't in my opinion.

Reply:
That is not the scope of this PR. This PR is documentation.

Reply:
https://github.com/tue-robotics/hero_bringup/issues/41